### ISSUE TITLE
Create YAML file for Prithvi Fine-Tuning on BioMassters

### ISSUE THREAD

**Issue by DWGodwin** [ClarkCGA/gfm-biomassters#1]
The first step in running the experiments using TerraTorch is to create a YAML which directs TerraTorch to perform the fine-tuning on the BioMassters dataset. The simplest case will be used, in which Sentinel-2 is the only data.

**Comment by DWGodwin** [comment]
Transforms are working appropriately and model loss is decreasing! We are now able to conduct the original experiments using S2. Next step is to create a config for multimodal experiments - then, we can begin to set up and re-do the experiments in Terratorch.

**Comment by DWGodwin** [comment]
The config for multi-modal experiments is functional, and trainings for both S2 and S2/S1 multimodal experiments ran for 19 hours before Blade had to be restarted due to networking issues. The loss curves for both decrease during this time, but I think the identity decoder I am using as a test is not best for pixel-level regression. I will re-do experiments with the FCN decoder, which is more analogous to the original experiments, and we will see how it does with 4 time steps.

Here is a plot of the multimodal validation RMSE:
<img width="347" height="254" alt="Multimodal Validation RMSE" src="https://github.com/user-attachments/assets/4ff9232e-3ca5-4567-9bb0-692b852c4889" />

Here is a plot of the S2 only validation RMSE:
<img width="336" height="256" alt="S2 Validation RMSE" src="https://github.com/user-attachments/assets/ae7af28b-63c2-42fa-8b8a-d8c6401f0c2c" />

Here is a typical output and target, with an RGB visualization of the first time step - both experiments are equivalently blurry but show that the model is fitting the data:
<img width="797" height="272" alt="Image" src="https://github.com/user-attachments/assets/434de1b9-30b4-4326-98bc-8611467ee1a1" />

**Comment by DWGodwin** [comment]
I have created a config yaml which uses a Fully Convolutional Network as the decoder. The results from Terratorch are on par with results from earlier this year.

Here is an example RMSE curve during training from the multimodal model as well as an example prediction:

<img width="352" height="251" alt="Image" src="https://github.com/user-attachments/assets/ac5b2679-eb46-4cf9-b695-106fe3661a3d" />

<img width="1017" height="385" alt="Image" src="https://github.com/user-attachments/assets/ed179749-3d9b-4e5f-a1a0-041fea5c2b2b" />

Oddly, the multimodal and S2-only models have similar RMSE scores after ~150K iterations.

Now that we a working config in Terratorch, we should create another round of experiments and collect the results.
